ecosystem/sep-0012.md

```diff
@@ -182,25 +196,38 @@ Name | Type | Description
 }
 ```
 
-#### Status
+#### Customer Statuses
 
 Status | Description
 -------|------------
-ACCEPTED | KYC has been accepted and the customer can be identified by stellar account and optional memo in future transactions.  It is possible for an accepted customer to move back to another status if the KYC provider determines it needs more info at a later date, or if the customer shows up on a sanctions list.
-PROCESSING | KYC process is in flight and client can check again in the future to see if any further info is needed
+ACCEPTED | All required KYC fields have been accepted. It is possible for an accepted customer to move back to another status if the KYC provider determines it needs more info at a later date, or if the customer shows up on a sanctions list.
+PROCESSING | KYC process is in flight and client can check again in the future to see if any further info is needed.
 NEEDS_INFO | More info needs to be provided to finish KYC for this customer.  The `fields` entry is required in this case.
 REJECTED | This customer's KYC has failed and will never succeed.  The `message` must be supplied in this case.
 
 #### Fields
 
-The fields entry is required for the `NEEDS_INFO` status.  Fields should be specified as an object with keys representing field names required, preferably from SEP-9.
+The fields entry is required for the `NEEDS_INFO` status but may be included with any status. Fields should be specified as an object with keys representing the [SEP-9](sep-0009.md) field names required.
 
 Property | Type | Description
 -----|------|------------
 `type` | enum | The data type of the field value. Can be `string`, `binary`, `number`, or `date`
 `description` | string | A human-readable description of this field, especially important if this is not a SEP-9 field.
 `choices` | array | (optional) An array of valid values for this field.
 `optional` | boolean | (optional) A boolean whether this field is required to proceed or not. Defaults to false.
+`status` | string | (optional) One of the values described in [Field Statuses](#field-statuses).
```

Comment:
❓ I like that this field is optional because in some jurisdictions the server may wish to state that the customer is accepted or rejected without indicating which fields caused the decision. Is it acceptable for a server to implement SEP-12 and not output the status? Or is the optionality only to accommodate implementations that existed prior to this change?

Reply:
I made it optional to accommodate existing implementations, but you make a good point, the field could be omitted for that reason as well.

Reply:
I think it could be worth making that more explicit, by adding a statement that servers may choose not to expose the status per-field status if they don't wish expose which fields were accepted or rejected. Does the protocol still work in this case do you think?

Reply:
I think it still works. Anchors who don't want to expose `accepted`/`rejected` information can omit the `status` attribute unless verification is needed. I'll add a statement.